target/linux/ramips/image/mt7621.mk

```diff
@@ -610,6 +610,15 @@ define Device/gnubee_gb-pc2
 endef
 TARGET_DEVICES += gnubee_gb-pc2
 
+define Device/hilink_hlk-7621a-evb
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  DEVICE_VENDOR := HiLink
+  DEVICE_MODEL := HLK-7621A EVB
```

Comment:
Please make this consistent with the "model" in DTS.

Reply:
Will change into  "HLK-7621A evaluation board" as it is in dts file.